dali/daliadmin/daliadmin.cpp

```diff
@@ -55,49 +55,53 @@ void usage(const char *exe)
   printf("  count <xpath>              -- counts xpath matches\n");
   printf("\n");
   printf("Logical File meta information commands:\n");
-  printf("  dfsfile <logicalname>          -- get meta information for file\n");
-  printf("  dfsmeta <logicalname> <storage> -- get new meta information for file\n");
-  printf("  setdfspartattr <logicalname> <part> <attribute> [<value>] -- set attribute of a file part to value, or delete the attribute if not provided\n");
-  printf("  dfspart <logicalname> <part>   -- get meta information for part num\n");
+  printf("  checksuperfile <superfilename> [fix=true|false] -- check superfile links consistent and optionally fix\n");
+  printf("  checksubfile <subfilename>     -- check subfile links to parent consistent\n");
+  printf("  checkfilesize <logicalfilemask> [fix=true|false] -- check file size attributes and optionally fix");
```

Comment:
NB: this is the new command, the other changes here are only alphabetic reordering